src/H5VLint.c

```diff
@@ -987,7 +987,7 @@ H5VL_conn_dec_rc(H5VL_t *connector)
     connector->nrefs--;
 
     /* Check for last reference */
-    if (0 == connector->nrefs) {
+    if (0 >= connector->nrefs) {
```

Comment:
This doesn't look correct to me.  We should only decrement the refcount on the ID when the connector struct is no longer using it.

Reply:
Would it be better to replace this change with an assertion that the ref count should be greater than zero?

Reply:
Agree, but that may need to be handled at the source (in the code which increments and doesn't decrement).

Reply:
I'm not sure the change addressed this.